modules/menu/menu.c

```diff
@@ -881,6 +885,7 @@ static void event_handler(enum ua_event ev, struct bevent *event, void *arg)
 		if (call_is_outgoing(call))
 			--menu.outcnt;
 
+		play_hangup(call);
```

Comment:
There is a collision with `menu_play_closed()`. We should discuss
- Currently if the current call terminates and there is an `scode`, then an error is played. Then the `hangup_aufile` should not be played in addition.
- What should be the expected behavior if multiple calls are handled?

Reply:
>Currently if the current call terminates and there is an scode, then an error is played. Then the hangup_aufile should not be played in addition.

The hangup file is now only played when there is no sip error